articles/add-user-info-from-idp-to-host.md

```diff
@@ -0,0 +1,284 @@
+# Add user information from identity provider (IdP) to host
+
+![Import users from IdP to Fleet](../website/assets/images/articles/add-users-from-idp-cover-img.png)
+
+_Available in Fleet Premium._
+
+Learn how to connect your identity provider (IdP) to retrieve end-user information and map it to a
+host. Fleet uses [SCIM](https://scim.cloud/) protocol.
+
+Below you can learn how to connect [Okta](https://www.okta.com/), [Microsoft Entra ID](https://www.microsoft.com/en-us/security/business/identity-access/microsoft-entra-id) or [Authentik](https://goauthentik.io/).
```

Comment:
```suggestion
Fleet supports [Okta](#okta), [authentik](#authentik), [Microsoft Active Directory (AD) / Entra ID](#microsoft-entra-id), as well as any other identity provider (IdP) that supports the [SCIM](https://scim.cloud/) standard.

To add IdP host vitals, follow steps for your IdP.
```

I think we can make this language consistent with what we have here in the [SSO docs](https://fleetdm.com/docs/deploy/single-sign-on-sso#single-sign-on-sso):

![Screenshot 2025-03-21 at 9 35 56 AM](https://github.com/user-attachments/assets/3b3526a5-ecf0-4217-9603-18adcf076bbb)